proxy/jsonrpc/client.go

```diff
@@ -31,6 +32,12 @@ type API struct {
 	}
 }
 
+// GetIDsResult is returned by `GetIDs` method.
+type GetIDsResult struct {
```

Comment:
why is this re-declaration needed? there is already one in the da package?

Reply:
Good catch! I hoped that I will somehow hide this and use only for jsonrpc, but later I had to introduce it also in da.